nabla-pendulum/deploy/06_deploy_swap_pools.ts

```diff
@@ -0,0 +1,75 @@
+import { WasmDeployEnvironment } from "../../src/index";
+import { registerSwapPool } from "../_lib";
+
+async function DeploySwapPools({ getNamedAccounts, deployments }: WasmDeployEnvironment) {
+  const { deployer } = await getNamedAccounts();
+  const treasury = deployer;
+
+  const [backstop, router, curveLowSlippage, curveNoSlippage, usdc, eurc, brl] = await Promise.all([
+    deployments.get("backstop"),
+    deployments.get("router"),
+    deployments.get("nablaCurveLowSlippage"),
+    deployments.get("nablaCurveNoSlippage"),
+    deployments.get("usdcErc20Wrapper"),
+    deployments.get("eurcErc20Wrapper"),
+    deployments.get("brlErc20Wrapper"),
+  ]);
+
+  const poolUsdt = await deployments.deploy("swap-usdc", {
+    from: deployer,
+    contract: "SwapPool",
+    args: [
+      usdc.address,
+      curveNoSlippage.address,
+      router.address,
+      backstop.address,
+      treasury.accountId,
+      "USDC Swap LP",
+      "USDC-LP",
+    ],
+    log: true,
+  });
+
+  const poolEurc = await deployments.deploy("swap-eurc", {
+    from: deployer,
+    contract: "SwapPool",
+    args: [
+      eurc.address,
+      curveLowSlippage.address,
+      router.address,
+      backstop.address,
+      treasury.accountId,
+      "EURC Swap LP",
+      "EURC-LP",
+    ],
+    log: true,
+  });
+
+  const poolBrl = await deployments.deploy("swap-brl", {
+    from: deployer,
+    contract: "SwapPool",
+    args: [
+      brl.address,
+      curveLowSlippage.address,
+      router.address,
+      backstop.address,
+      treasury.accountId,
+      "BRL Swap LP",
+      "BRL-LP",
+    ],
+    log: true,
+  });
+
+  await registerSwapPool(deployments, { from: deployer, log: true }, usdc.address, poolUsdt.address, 100);
```

Comment:
Is the insurance fee supposed to denote 1% (according to [this](https://www.notion.so/satoshipay/24-05-16-Nabla-Pendulum-Deployment-c7cefd58f44c428a92ceaea30c603286?pvs=4#006d449e918340afbbcda16acb354c81) doc) here or is it an absolute value?

Reply:
Nevermind, I just realized that this parameter is called `insuranceBps`, so indeed it means 1%